e2e-tests/cypress/integration/guidance-page-before-you-appeal.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Guidance Page - Before you Appeal
+  As a PO on the appeal service
+  I want prospective appellants to be aware of who the service is for
+  So that only those who need to appeal through the new appeal service do so
+
+  Scenario: Appellant navigates to next page
```

Comment:
It should be "
  Scenario: AC01 - Appellant navigates to next page"

Reply:
Team 1 don't seem to mark their Scenarios like this so I was maintaining consistency. Should we treat this differently?

Reply:
as far as I know, they had started following this suite later on in development. We have been maintaining this from beginning.